NDTensors/ext/NDTensorMetal/adapt.jl

```diff
@@ -0,0 +1 @@
+@inline NDTensors.mtl(xs; unified::Bool=false) = NDTensors.adapt_structure(MtlArray, xs)
```

Comment:
Should this just be `adapt(MtlArray, xs)` instead of `adapt_storage(MtlArray, xs)`?

Reply:
Also seems like `unified` isn't needed/used here.